api/server.js

```diff
@@ -0,0 +1,37 @@
+require("dotenv").config();
+const express = require("express");
+const helmet = require("helmet");
+const logger = require("morgan");
+
+const postsRouter = require("../posts/postRouter");
+const userRouter = require("../users/userRouter");
+
+const server = express();
+
+// Global Middleware
```

Comment:
Like I said before; this is a good way of organizing everything. Very readable 👍